assets/js/index.js

```diff
@@ -0,0 +1,142 @@
+// global
+const workHours = [
+  { time: "9:00", key: 9 },
+  { time: "10:00", key: 10 },
+  { time: "11:00", key: 11 },
+  { time: "12:00", key: 12 },
+  { time: "13:00", key: 13 },
+  { time: "14:00", key: 14 },
+  { time: "15:00", key: 15 },
+  { time: "16:00", key: 16 },
+  { time: "17:00", key: 17 },
+];
+
+const header = $(".jumbotron");
+
+const main = $("#main");
+
+const currentHour = moment().hour();
+
+// initalise LS
+const onReady = () => {
+  // check if tasks exists in LS
+  const tasks = readFromLocalStorage();
+  // if false then set tasks to empty object in LS
+  if (!tasks) {
+    localStorage.setItem("tasks", JSON.stringify({}));
+  }
+
+  renderPage();
+};
+
+// get tasks from local storage
+const readFromLocalStorage = () => {
+  // get from LS by key
+  const getTasks = localStorage.getItem("tasks");
+  // parse LS data
+  const parsedData = JSON.parse(getTasks);
+  return parsedData;
+};
+
+const renderDate = () => {
+  // append date to header
+  header.append(
+    `<p id="currentDay" class="lead">${moment().format(
+      "dddd Do MMMM YYYY"
+    )}</p>`
+  );
+};
+
+const renderTime = () => {
+  // append time to header
+  header.append(
+    `<p id="currentTime" class="lead">${moment().format("H:mm")}</p>`
+  );
+};
+
+const getClassName = (hour) => {
+  // if time block hour is before now, return past
+  if (hour < currentHour) {
+    return "past";
+    // if time block hour is after now, return future
+  } else if (hour > currentHour) {
+    return "future";
+    // leave current hour as-is
+  } else {
```

Comment:
If you find a code block is empty like this, it would suggest that it isn't needed. If you don't need a class name for the present hour, you can just `return ""`